package.json

```diff
@@ -32,7 +32,9 @@
     "gh-actions-build": "./.github/scripts/buildActions.sh",
     "gh-actions-validate": "./.github/scripts/validateActionsAndWorkflows.sh",
     "analyze-packages": "ANALYZE_BUNDLE=true webpack --config config/webpack/webpack.common.js --env.envFile=.env.production",
-    "check-metro-bundler-port": "node config/checkMetroBundlerPort.js"
+    "check-metro-bundler-port": "node config/checkMetroBundlerPort.js",
+    "symbolicate:android": "npx metro-symbolicate android/app/build/generated/sourcemaps/react/release/index.android.bundle.map",
```

Comment:
Are these scripts used by react native automatically?

Reply:
> Are these scripts used by react native automatically?

I'm not sure what do you mean - how would they be used automatically by react-native?

I've outlined sample usage in the "Details" section. 

Something like this:
1. Build pipeline runs and saves sourcemaps somewhere
2. Firebase Crashlytics captures a crash
3. Something uses crash report data to retrieve the source maps for the build (version)
4. And runs `npm run symbolicate:{platform} < crashlytics.stactrace.txt` where `.stacktrace.txt` are js stack traces from "Firebase Crashlytics"

Reply:
Awesome, thanks for the explanation. Sounds like a good plan